conngen/conngenmodule.cpp

```diff
@@ -281,23 +264,32 @@ ConnectionGeneratorModule::CGSelectImplementation_s_sFunction::execute(
    purposes. Usually, connections are created from a
    ConnectionGenerator using CGConnect.
 
+   Availability: Only if compiled with libneurosim support
```

Comment:
Applies to lines 258f above: doc says `sources`, `targets` must be lists, but shouldn' that be GIDCollections?